init/src/main/groovy/com/taulia/boundaries/Transmission.groovy

```diff
@@ -2,9 +2,22 @@ package com.taulia.boundaries
 
 
 class Transmission {
+
   Gear gear
   Moveable moveable
+
   Transmission(Moveable moveable){
     this.moveable = moveable
   }
+
+  void shift(Gear gear) {
+    if (canShiftToGear(gear)) {
+      this.gear = gear
+    }
+  }
+
+  private boolean canShiftToGear(Gear gear) {
```

Comment:
I like this, pulling it out makes it more testable, and the intent is clearer too.  Great Job.